docs/versioned/flink/flink-getting-started.md

```diff
@@ -547,6 +547,33 @@ RewriteDataFilesActionResult result = Actions.forTable(table)
 
 For more doc about options of the rewrite files action, please see [RewriteDataFilesAction](../../../javadoc/{{% icebergVersion %}}/org/apache/iceberg/flink/actions/RewriteDataFilesAction.html)
 
+## Configuration.
+
+The following options can be used to tune the performance of the query execution.
+
+| Key                                             | Default | Type    | Description                                                                                                                      |
+|:-----------------------------------------------:| ------- | ------- | -------------------------------------------------------------------------------------------------------------------------------- |
+| table.exec.iceberg.infer-source-parallelism     | True    | Boolean | If is false, parallelism of source are set by config.<br/>If is true, source parallelism is inferred according to splits number. |
+| table.exec.iceberg.infer-source-parallelism.max | 100     | Int     | Sets max infer parallelism for source operator.                                                                                  |
+| table.exec.iceberg.expose-split-locality-info   | None    | Boolean | Expose split host information to use Flink's locality aware split assigner.                                                      |
+
+Usage:
```

Comment:
I think we need to add two cases for the usage: 

1.  How to configure it in SQL client ?
2. How to configure it in Flink Java DataStream job ?

Reply:
Added the case for the SQL client.
for Java DataStream job, i refined the second use case as follows:
```java
StreamExecutionEnvironment env = StreamExecutionEnvironment.createLocalEnvironment();
TableLoader tableLoader = TableLoader.fromHadoopTable("hdfs://nn:8020/warehouse/path");
    
Configuration configuration = new Configuration();
configuration.setBoolean(FlinkConfigOptions.TABLE_EXEC_ICEBERG_INFER_SOURCE_PARALLELISM, false);
DataStream<RowData> source = FlinkSource.forRowData()
    .env(env)
    .tableLoader(tableLoader)
    .flinkConf(configuration)
    .streaming(true)
    .build();
```
Is it okay to write this way?
thx.

Reply:
@hililiwei I added some sample code in the Javadoc for the `FlinkConfigOptions` class

https://github.com/apache/iceberg/pull/4060/files#diff-cb10b0c952e576ba625a982a097f0442fb39c4b26e273c69ebf24204e42356ce

Reply:
Great. Am I quoting javadoc here, or a hyperlink to javadoc? Or if we don't need to add document, I can close this PR.